.gitlab-ci.yml

```diff
@@ -343,6 +338,7 @@ integration:docker:
     DOCKER_TLS_CERTDIR: "/certs"
     FF_NETWORK_PER_BUILD: "true"
     PK_TEST_TMPDIR: "${CI_PROJECT_DIR}/tmp/test"
+    PK_NETWORK: "testnet"
```

Comment:
Ok so that's for the tests, but if we switch to using fully qualified names, then `testnet.polykey.com` would be the correct designation.

Reply:
This will need to be made into a new issue and addressed in `Polykey` and `Polykey-CLI` later. It requires a few changes.

Reply:
Created a new issue for this at https://github.com/MatrixAI/Polykey-CLI/issues/97